h/util/markdown.py

```diff
@@ -27,7 +26,28 @@
     "strong",
     "ul",
 ]
-ALLOWED_TAGS = set(bleach.ALLOWED_TAGS + MARKDOWN_TAGS)
```

Comment:
I've pushed a lot of the "constants" here to the bottom where they are used. They aren't really constants in any meaningful way, and should probably be pushed inside the `_get_cleaner()` function where they are really just steps to do.

I've left the allowed tags as you might conceivably actually want to change that.

Reply:
Yeah, I'm not a fan of sticking a bunch of `CONSTANT`'s at the top of the file when you don't really need to. `ALLOWED_TAGS` is only used once so it's just not necessary